index.rst

```diff
@@ -260,92 +291,98 @@ If working with an individual CSC, which as an operator would be a rare occurren
+On the other hand, deriving new parameters may involve generating complex LUTs that may require on sky observations and detailed data analysis.
 
-Changing the default configuration is a more involved endeavor because it entails making a change to the contents of the configuration repository.
-Because the repo is under version control, the appropriate steps must be taken.
-For this example, let's assume we want to change the default configuration in the ATAOS, which is found in the `ATAOS directory of the ts_config_attcs repo <https://github.com/lsst-ts/ts_config_attcs/tree/develop/ATAOS>`__.
+Following is a detail of each step of the process to generate a new configuration and update it for CSCs written in salobj.
+For other components, see the exception section below.
 
 
-#.  Create a JIRA ticket in where the title/description note the change being made.
-    Let's assume it creates ticket DM-12345.
+#.  Create a Jira ticket to track the work being done (e.g. DM-12345).
+    If details or discussions are needed they can done using the Jira tickets itself.
 
-#.  Clone the repo and checkout a new branch
+    .. prompt:: bash
 
-    ::
+        git clone git@github.com:lsst-ts/ts_config_attcs.git
+        git checkout -b tickets/DM-12345
 
-      git clone git@github.com:lsst-ts/ts_config_attcs.git
-      git checkout -b tickets/DM-12345
 
-    Note that the branch name is the word ``tickets/`` appended with the Jira ticket name.
+#.  Execute the work needed to derive the new configuration parameter(s).
 
-#.  Open the most recent schema version (v2) and modify the contents of ``_labels.yaml``.
-    For example, the original version may be:
+    As mentioned above, in some cases, the process may be straightforward, consisting simply of replacing the values of a set of parameters with given values (e.g., swapping filters).
+    In these cases, this step will be simply verifying any required work was performed and continuing to the next step.
+    Jira should be used to track those activities.
 
-    ::
+    The Jira ticket should also be used to track the work done on those cases where a more involved analysis is required, e.g., in-dome and/or on-sky data acquisition, EFD queries, data processing etc.
+    Any ancillary software or data product required during this process should be properly managed using git.
+    When working with Telescope and Site components, any software required during this process should be stored in a git repository in `T&S GitHub organization <https://github.com/orgs/lsst-ts>`__, and should follow the standard `T&S development workflow guidelines <https://tssw-developer.lsst.io>`__.
+    This includes, but is not limited to, EFD queries, Jupyter notebooks, other data analysis routines (regardless of the programming language) and so on.
+    The preferred location for storing Jupyter notebooks is the `ts_notebooks <https://github.com/lsst-ts/ts_notebooks>`__ repository.
```

Comment:
Maybe add an item about "tech notes"? Something like:

```
If the procedure to generate the new configuration requires detailed explanations, a tech-note in tstn repository can be created.
```

Reply:
I tweaked it a tad from what you suggested.